src/coverage.ts

```diff
@@ -67,9 +77,17 @@ export const coverage = async ({
         description: `${percentString(rate)} coverage`
       }
     }
+    const state = evaluateState([
+      rate * 100 >= threshold,
+      Math.abs(diff * 100) >= differenceThreshold // absolute value because it is diff
+    ])
+    const failDescription =
+      state === 'success'
+        ? ''
+        : `Failed to satisfy threshold:${threshold}% and difference-threshold:${differenceThreshold}%`
```

Comment:
```suggestion
    const success = evaluateState([
      rate * 100 >= threshold,
      Math.abs(diff * 100) >= differenceThreshold // absolute value because it is diff
    ])
    const failDescription =
      success
        ? ''
        : `Failed to satisfy threshold:${threshold}% and difference-threshold:${differenceThreshold}%`
```

Reply:
applied